animeone/lib/ui/component/ErrorButton.dart

```diff
@@ -13,31 +13,25 @@ class ErrorButton extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    String finalMsg = '404 無法加載\n\n$msg)';
-    return Column(
-      mainAxisAlignment: MainAxisAlignment.center,
-      children: <Widget>[
-        Text(finalMsg, textAlign: TextAlign.center),
-        this.renderFixButton(),
-      ],
+    String finalMsg = '404 無法加載\n\n${msg ?? ''}';
+    return Center(
+      child: Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: <Widget>[
+          Text(finalMsg, textAlign: TextAlign.center),
+          renderFixButton(context),
```

Comment:
The button shouldn't be here all the time. A check if probably needed. Sometimes, it is just a very simple Internet error.